backend/common/src/main/java/ai/verta/modeldb/common/futures/FutureRest.java

```diff
@@ -0,0 +1,28 @@
+package ai.verta.modeldb.common.futures;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+
+@SuppressWarnings({"squid:S100"})
```

Comment:
what is this warning that's being suppressed?

Reply:
it is the SonarQube warnings about method naming convention. it was addressed there because we have a method name like `ServerResponse` and it was started with capitalizing later so added there. but now we have modified the name so no more needed it. I will remove it.